perf-test/app/page.tsx

```diff
@@ -222,11 +222,15 @@ export default function Home() {
 
     for (let i = 0; i < _count; i++) {
       const _anim = animations[Math.floor(Math.random() * animations.length)];
+      const lottieURL = `${urlPrefix}${_anim}`;
+
+      const response = await fetch(lottieURL);
```

Comment:
this fetching will not affect the loading time?

Reply:
It affects the loading time and light house score, will fix it soon. Thanks!

Reply:
Size is not quite necessary, we can just remove the type.